src/index.js

```diff
@@ -223,6 +224,35 @@ export default ({ apiKey = null, client = null } = {}) => {
                 })
                 .catch(callback);
         },
+        
+        /**
+         * create write stream
+         *
+         * @param {String} token
+         * @param {String} remotePath
+         * @returns {Stream}
+         */
+        createWriteStream(filepath) {
+            return dropboxStream.createDropboxUploadStream({
+                token: client.accessToken,
+                filepath,
+                chunkSize: 1000 * 1024
+            });
+        },
+
+        /**
+         * create read stream
+         *
+         * @param {String} token
+         * @param {String} remotePath
+         * @returns {Stream}
+         */
+        createReadStream(filepath) {
```

Comment:
Could be added the `options` argument, speciall the `start` and `end` fields?

Reply:
yup!

Reply:
`dropbox-stream` doesn’t support `start` and `end`

Reply:
Then time to add support for it :-)

Reply:
I would personally suggest to add those niceties in a separate PR to not block this one. I would argue that their usage is uncommon.

Reply:
Yup Im going to merge this :) thanks all

Reply:
> I would personally suggest to add those niceties in a separate PR to not block this one

Ok, I agree.